src/main/frontend/src/components/aggregates/AggregateDialog.svelte

```diff
@@ -89,12 +89,17 @@
 		}
 	}
 
+	const isAggregateNameUnique = (value) => {
+		const result = $aggregateSettings.filter(aggregate => aggregate.aggregateName === value);
```

Comment:
`.find` would be enough, I guess. Unique could be refactored out on all these, if you want to (function giving back a function).

Reply:
`.find` returns the first match, but I need to check if the value occurs more than one.
I tried closure(function return function) but it breaks error handling.

Reply:
`.find` return the first match, I want to show the error message if the value occurs more than 1. If I use `.find` I cannot be sure it's used multiple times or once

Reply:
fixed function